Lib/fontTools/feaLib/location.py

```diff
@@ -0,0 +1,9 @@
+class FeatureLibLocation(object):
```

Comment:
you could use a namedtuple with a custom `__str__` method, e.g. like this https://github.com/fonttools/fonttools/blob/80dac9c0f4304b70d2a8103b5962439d842e0ab9/Lib/fontTools/voltLib/ast.py#L2-L11